cvat/schema.yml

```diff
@@ -7385,6 +8171,170 @@ components:
         * `AZURE_CONTAINER` - AZURE_CONTAINER
         * `GOOGLE_DRIVE` - GOOGLE_DRIVE
         * `GOOGLE_CLOUD_STORAGE` - GOOGLE_CLOUD_STORAGE
+    QualityReport:
+      type: object
+      properties:
+        id:
+          type: integer
+          readOnly: true
+        job_id:
+          type: integer
+          nullable: true
+          readOnly: true
+        task_id:
+          type: integer
+          nullable: true
+          readOnly: true
+        parent_id:
+          type: integer
+          nullable: true
+          readOnly: true
+        target:
+          $ref: '#/components/schemas/QualityReportTarget'
+        summary:
+          $ref: '#/components/schemas/QualityReportSummary'
+        parameters:
+          $ref: '#/components/schemas/QualityReportParameters'
+        created_date:
+          type: string
+          format: date-time
+          readOnly: true
+        target_last_updated:
+          type: string
+          format: date-time
+          readOnly: true
+        gt_last_updated:
+          type: string
+          format: date-time
+          readOnly: true
+      required:
+      - parameters
+      - summary
+      - target
+    QualityReportCreateRequest:
+      type: object
+      properties:
+        task_id:
+          type: integer
+          writeOnly: true
+      required:
+      - task_id
+    QualityReportParameters:
+      type: object
+      properties:
+        iou_threshold:
+          type: number
+          format: double
+      required:
+      - iou_threshold
+    QualityReportSummary:
+      type: object
+      properties:
+        frame_count:
+          type: integer
+        frame_share_percent:
```

Comment:
Percentages are IMO strictly a display format. Raw data should use fractions (from 0 to 1).

Reply:
Done